Resources/Prototypes/Entities/Mobs/Player/dwarf.yml

```diff
@@ -5,6 +5,11 @@
   id: MobDwarf
   description: A miserable pile of secrets.
   components:
+    - type: InteractionPopup
+      successChance: 0.9
```

Comment:
should honestly be higher.

Reply:
I wanted a funny "haha you failed a hug and smacked the person on their head". Upping it to 1.0 would make it 100% success rate, but I can make that change if you want :D